### PR TITLE
Expanded/clarified introduction

### PR DESCRIPTION
**Pull request by patrickhlauke**
Probably very subjective, but...perhaps this introduction makes the
intention slightly clearer?

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -141,12 +141,14 @@
 
       <h2>Introduction</h2>
 
-      <p>The Pointer Lock API provides for input methods of applications based on
-      the movement of the mouse, not just the absolute position of a cursor.  A
+      <p>The Pointer Lock API provides applications with the ability to directly interpret mouse movements
```
Comment (scheib) [inline index.html:144]: "with the ability" -> "the ability"

Comment (patrickhlauke) [inline index.html:144]: "provide with" seems more appropriate in this case (see http://www.oxforddictionaries.com/definition/english/provide example "A submarine fleet is still essential in providing Canada with the necessary means to defend the world's longest coastline"). but perhaps this is a en-gb vs en-us thing?

Comment (scheib) [inline index.html:144]: Thanks for the contribution. Let's not debate non-normative style.

**Comment by siusin** [comment]
Marked as non-substantive for IPR from ash-nazg.

**Pull request merged**